terraform/cloud-platform-components/external-dns.tf

```diff
@@ -16,8 +16,12 @@ resource "aws_iam_role" "external_dns" {
 
 data "aws_iam_policy_document" "external_dns" {
   statement {
-    actions   = ["route53:ChangeResourceRecordSets"]
-    resources = ["arn:aws:route53:::hostedzone/${data.terraform_remote_state.cluster.hosted_zone_id}"]
+    actions = ["route53:ChangeResourceRecordSets"]
+
+    resources = [
+      "arn:aws:route53:::hostedzone/${data.terraform_remote_state.cluster.hosted_zone_id}",
```

Comment:
When that ternary is false, this will contain the same zone id twice. Can we not use an empty string for false?

Reply:
couldn't find a way, terraform insist on sending an empty pair of quotes that breaks indenting

Reply:
Do you mean it breaks validation with the AWS API?
We could do something like this then:
```
["${compact(list(
      "arn:aws:route53:::hostedzone/${data.terraform_remote_state.cluster.hosted_zone_id}",
      "${terraform.workspace == local.live_workspace ? format("%s/%s", "arn:aws:route53:::hostedzone", data.terraform_remote_state.global.cp_zone_id) : ""}",
    ))}"]
```
But I'm not sure this reads better 🍝 
It's just that the fact that the duplicates get pruned when you actually apply is perhaps a "feature" we shouldn't depend upon (couldn't find any mention of this) 🤷‍♂️